charts/nri-prometheus/README.md

```diff
@@ -12,6 +12,7 @@ This chart will deploy the New Relic's Prometheus OpenMetrics Integration.
 | `global.licenseKey` - `licenseKey`                         | The [license key](https://docs.newrelic.com/docs/accounts/install-new-relic/account-setup/license-key) for your New Relic Account. This will be preferred configuration option if both `licenseKey` and `customSecret` are specified. |                                        |
 | `global.customSecretName` - `customSecretName`             | Name of the Secret object where the license key is stored                                                                                                                                                                             |                                        |
 | `global.customSecretLicenseKey` - `customSecretLicenseKey` | Key in the Secret object where the license key is stored.                                                                                                                                                                             |                                        |
+| `global.lowDataMode` - `lowDataMode` | Excludes KSM and cAdvisor metrics when enabled overwriting config.transformations                                                                                                                                                                                                      |           `false`                      |
```

Comment:
I think this needs to be a bit more clear. What do you think about:

Excludes KSM and cAdvisor metrics when enabled. The low data mode overwrites `config.transformations` and `config.require_scrape_enabled_label_for_nodes`, if you want to set these fields in the config you need to turn low data mode off.

Reply:
I agree, I am removing `require_scrape_enabled_label_for_nodes` only because it is true by default and there is no reason to fix it in the static file